### DISCUSSION TITLE
How can I float and resize an image using Markdown editor

### DISCUSSION THREAD

**Discussion by jord8on** [requarks/wiki#3432] [Help / Questions]
**Question**
In wiki.js I can't figure out how to float an image using the markdown editor.

I saw this issue, but it looks like that's related to the WYSIWYG editor: https://github.com/Requarks/wiki/issues/1413 

```
<div style="max-width: 50%; float: right;">
 ![image.png](/folder/image.png)</div>
```

I tried that ⬆︎  and it floats everything between the divs, to the right, but it doesn't render the image.

Is there some library/index of all the parameters/modifiers that can be used with this markdown editor?

I use Discourse forums and they have a markdown editor. I can use the following syntax to float an image to the right, and also resize the originall 499x499 image to be 120px wide.

`[floatr]![image|499x499,120x](upload://image.png)[/floatr]`

Thanks in advance for any insight here!

**Host Info (please complete the following information):**
OS: [e.g. Ubuntu 18.04, Docker]
Wiki.js version: [e.g. 2.4.107]
Database engine: [e.g. PostgreSQL 11.8 Node.js 12.18.0]

**Comment by NGPixel** [comment]
You can't mix Markdown and HTML on the same line.

This should work:
```
<div style="max-width: 50%; float: right;">

![image.png](/folder/image.png)

</div>
```
